src/cache.c

```diff
@@ -35,26 +35,48 @@ static pthread_mutex_t cf_lock;
  */
 static char *DATA_DIR;
 
-static char *CacheSystem_get_cache_home()
+
+char *CacheSystem_get_cache_dir()
 {
+    const char *default_cache_subdir = "/.cache";
+    char *cache_dir;
+
     if (CONFIG.cache_dir) {
         return CONFIG.cache_dir;
+    } else {
+        /*
+        *  "XDG_CACHE_HOME/HOME/.cache" > "HOME/.cache"  > "./.cache"
+        */
```

Comment:
I am not sure if the comment reflects what's actually happening. The code is clear enough describing what's happening. Perhaps delete this?